turbinia/lib/google_cloud.py

```diff
@@ -24,23 +24,53 @@
 
 from turbinia import TurbiniaException
 
+import datetime
 
-def setup_stackdriver_handler(project_id):
+from google.cloud.logging import _helpers
+from google.cloud.logging.handlers.transports.background_thread import _Worker
+
+
+def setup_stackdriver_handler(project_id, environment):
   """Set up Google Cloud Stackdriver Logging
 
   The Google Cloud Logging library will attach itself as a
   handler to the default Python logging module.
 
   Attributes:
     project_id: The name of the Google Cloud project.
+    environment: Where the log is running.
```

Comment:
maybe make this a little more explciti?  "Logging origin - 'server' or 'psqworker'"

Reply:
fixed.